lib/components/type_ahead.ex

```diff
@@ -43,7 +44,8 @@ defmodule CrunchBerry.Components.TypeAhead do
           <li class="w-full px-2 py-3 cursor-pointer hover:bg-blue-3 hover:text-white <%=is_focus?(idx, assigns) %>"
               phx-click="type-ahead-select"
               phx-value-type-ahead-result-id="<%= id %>"
-              phx-value-type-ahead-result="<%= result %>">
+              phx-value-type-ahead-result="<%= result %>"
+              phx-target="<%= assigns.target %>">
```

Comment:
When `assigns.target` isn't provided, you don't want to render `phx-target` at all, since `phx-target=""` will cause events to be dropped. I'd probably move this into a helper

```elixir
def render(assigns) do
  ~L"""
    <li ... <%= phx_target(assigns) %>>
  """
end

defp phx_target(%{target: target}), do: ~s(phx-target="#{target}")
defp phx_target(_), do: nil
```

Reply:
Just going to update this for now so that the existing pages aren't broken.